docs/install/any-kubernetes-cluster.md

```diff
@@ -564,6 +571,14 @@ ConfigMap to specify which configurations are used for which namespaces:
            namespace: knative-eventing
    ```
 
+This Broker implementation has a small enough incremental overhead that it
+creates a Broker for every namespace by default.  To opt particular namespaces
+out of this behavior you can label them with:
+
+   ```bash
+   kubectl label namespace default knative-eventing-injection=disabled
```

Comment:
Is this really going to work? if you first create the NS, ns controller will create the broker? perhaps we should say here that you have to create with the label?

Reply:
You'd have to do it before installing things today.  Honestly, setting this label to "disabled" with EITHER namespace controller should clean up the auto-generated Broker, opened https://github.com/knative/eventing/issues/2796.